src/main/resources/org/nest/sympy/misc.py

```diff
@@ -0,0 +1,178 @@
+from sympy.parsing.sympy_parser import parse_expr
+from sympy import *
+
+from sympy.matrices import zeros
+
+# Define constants
+MAX_ORDER = 10
```

Comment:
I honestly don't think it is necessary to be allowed to have anything above 10. That would just be crazy. But I will add comments.